.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,77 @@
+name: 'GH Actions - CI'
```

Comment:
Some advice I read suggested matching this `name` value to the filename. So maybe change it to `name: 'Pull Request'` ?

Reply:
Or maybe change the filename to `gh-actions-ci.yml`, instead?

Reply:
I went with `gh-actions-pr` and updated the name in the file as well since it's targetted only at pull requests.